src/views/clusteroverview/OverviewTab/getting-started-card/quick-starts-section/utils.ts

```diff
@@ -44,7 +45,10 @@ export const orderQuickStarts = (
   // Show other not started quick starts (which are not featured)
   orderedQuickStarts.push(
     ...filteredQuickStarts.filter(
-      (q) => !isFeatured(q) && getStatus(q) === QuickStartStatus.NOT_STARTED,
+      (q) =>
+        !isFeatured(q) &&
+        (getStatus(q) === QuickStartStatus.NOT_STARTED ||
+          getStatus(q) === QuickStartStatus.COMPLETE),
```

Comment:
Probably want to also adjust the comment above to say "not started or complete".